torch/_inductor/codegen/cpp.py

```diff
@@ -59,6 +59,9 @@
     torch.int32: "int",
     torch.int16: "short",
     torch.int8: "signed char",
+    torch.uint64: "unsigned long",
```

Comment:
As far as I know, while in practice (on 64-bit systems) these types have the right bit-size, this is not guaranteed by the standard. Maybe it's better to use C++11 fixed width integer types? ( https://en.cppreference.com/w/cpp/types/integer )

Reply:
Yeah, I don't know why the other usages here weren't using the fixed width types lol